pkg/machines/app.jsx

```diff
@@ -22,10 +22,10 @@ import { AlertGroup, Alert } from '@patternfly/react-core';
 import { superuser } from "superuser.js";
```

Comment:
"machines: bring some order in the direstory structure and file naming " commit message could be fixed.
The title mistypes `directory`, you used `where` instead of `were` twice and spelling of `don't`

Reply:
that was horrible